ast.go

```diff
@@ -1058,7 +1094,22 @@ func (node *ColumnDef) String() string {
 		}
 		constraint = " " + strings.Join(constraints, " ")
 	}
-	return fmt.Sprintf("%s %s%s", node.Name, node.Type, constraint)
+	return fmt.Sprintf("%s %s%s", node.Column, node.Type, constraint)
+}
+
+// HasPrimaryKey checks if column definition has a primary key constraint.
+func (node *ColumnDef) HasPrimaryKey() bool {
```

Comment:
A helper method is [used in the grammar](https://github.com/tablelandnetwork/go-sqlparser/pull/29/files#diff-a6ec253416e8e9e6846f73f5c7677d6ff6fa85f5c37317f84e608fc50ff8238eR1008) to check if a column definition has a primary key.